scipy/sparse/linalg/_isolve/_gcrotmk.py

```diff
@@ -221,10 +221,10 @@ def gcrotmk(A, b, x0=None, *, rtol=1e-5, atol=0., maxiter=1000, M=None, callback
         According to [2]_, good values are around m.
         Default: m
     CU : list of tuples, optional
-        List of tuples ``(c, u)`` which contain the columns of the matrices
+        List of tuples `(c, u)` which contain the columns of the matrices
         C and U in the GCROT(m,k) algorithm. For details, see [2]_.
         The list given and vectors contained in it are modified in-place.
-        If not given, start from empty matrices. The ``c`` elements in the
+        If not given, start from empty matrices. The `c` elements in the
```

Comment:
This is not a argument to the function so was correct before
```suggestion
        If not given, start from empty matrices. The ``c`` elements in the
```